modules/installation-configuration-parameters.adoc

```diff
@@ -220,6 +220,22 @@ ifdef::osp[]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.openstack.rootVolume.size`
```

Comment:
@mandre :+1: or :-1: on these additions?

Reply:
Thanks for documenting this @maxwelldb. I've left some comments.